src/libs/OptionsListUtils.js

```diff
@@ -615,6 +615,7 @@ function getOptions(
         includeThreads = false,
         includeTasks = false,
         includeMoneyRequests = false,
+        disallowNewChats = false,
```

Comment:
Let's use exclude word to be consistent with others.
I suggest `excludeUnknownUsers`

Reply:
Agreed, changed it to your suggestion. Thanks!

Reply:
Thanks for the quick update. Also please pull main

Reply:
Thank you @situchan , done!

I pulled main before I made my PR, seems like a lot of stuff was merged afterwards